### ISSUE TITLE
Goal is deleted after shutdown

### ISSUE THREAD

**Issue by snipat** [timqian/one-goal#4]
Hello. When I quit the app or shutdown my laptop after typing a goal, the goal is lost and I have to type it again. Apart from that, great focusing app.

**Comment by macOS-Mavericks** [comment]
Yeah, it would be great to save the goal and the associated emoji.
